dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/imports/bundle/persister/DefaultTrackerObjectsDeletionService.java

```diff
@@ -194,6 +194,15 @@ public TrackerTypeReport deleteRelationships(TrackerBundle bundle) {
       org.hisp.dhis.relationship.Relationship relationship =
           relationshipService.getRelationship(uid);
 
+      relationship
+          .getTrackedEntityOrigins()
+          .forEach(
+              teUid -> {
+                TrackedEntity trackedEntity = teService.getTrackedEntity(teUid.getValue());
+                trackedEntity.setLastUpdatedByUserInfo(bundle.getUserInfo());
+                teService.updateTrackedEntity(trackedEntity);
```

Comment:
This means 2 requests to the DB x 1 (from) or 2 (from+to if bidirectional) TEs x number of relationship that are deleted 😅 I don't know how often relationships are deleted though.